pylint/__main__.py

```diff
@@ -2,6 +2,7 @@
 
 # Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
 # For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/pylint/graphs/contributors
```

Comment:
Do we want to replace this with a link to `CONTRIBUTORS.txt`?

Reply:
Yes, it's the old code (draft) before taking into account the remarks from the astroid MR :)